io.github.supercollider.SuperCollider.yml

```diff
@@ -0,0 +1,66 @@
+app-id: io.github.supercollider.SuperCollider
+runtime: org.kde.Platform
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-22.08'
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: scide
+rename-icon: supercollider
+rename-icon: sc_ide
+rename-desktop-file: SuperColliderIDE.desktop
+
+modules:
+  - name: SuperCollider
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DSC_EL=no
+    sources:
+      - type: archive
+        url: https://github.com/supercollider/supercollider/releases/download/Version-3.13.0/SuperCollider-3.13.0-Source.tar.bz2
+        sha256: 0fc5dba5baeae37f9069ad28885aa405c6a25309e3886cbe111bd3b7c05531ce
+  - name: fftw3
```

Comment:
you mean supercollide doesn't depend on fftw3? Also please use the one on shared-modules.